diff --git a/services/settings/dumps/gen_last_modified.py b/services/settings/dumps/gen_last_modified.py
index d16eaa01e3..0249bc2443 100644
--- a/services/settings/dumps/gen_last_modified.py
+++ b/services/settings/dumps/gen_last_modified.py
@@ -55,6 +55,7 @@ def main(output):
         "mobile/ios",
         "comm/mail",
         "comm/suite",
+        "bluegriffon",
     ), (
         "Cannot determine location of Remote Settings "
         f"dumps for platform {buildconfig.substs['MOZ_BUILD_APP']}"
@@ -72,6 +73,8 @@ def main(output):
         dumps_locations += ["comm/mail/app/settings/dumps/"]
     elif buildconfig.substs["MOZ_BUILD_APP"] == "comm/suite":
         dumps_locations += ["services/settings/dumps/"]
+    elif buildconfig.substs["MOZ_BUILD_APP"] == "bluegriffon":
+        dumps_locations += ["services/settings/dumps/"]

     remotesettings_dumps = {}
     for dumps_location in dumps_locations:
